docs/website/mmp-errors.md

```diff
@@ -776,6 +776,14 @@ You can silence this warning by adding `--nowarn=5221` to the **Additional mmp a
 
 ### MM5312: pkg-config failed with an error code '{code}'. Check build log for details.
 
+<a name="MM5313" />
+
+### MM5313: Could not find pkg-config. Please install the Mono.framework from https://mono-project.com/Downloads
```

Comment:
dupe of

`#### MM5301: pkg-config could not be found. Please install the Mono.framework from http://mono-project.com/Downloads`

sadly localization makes this harder to spot inside PR :(

Reply:
I ended up removing the MM5301 error message, because it's not used anymore in mmp, and the same error code is used for something else in mtouch.